From 787999b244144b79f2ea7c8f3df9917187c780aa Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Fri, 30 Mar 2012 17:52:03 +0200
Subject: [PATCH] use LIB_INSTALL_DIR

---
 bluedevil/bluedevil.pc.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bluedevil/bluedevil.pc.in b/bluedevil/bluedevil.pc.in
index 0d61c02..e6742bd 100644
--- a/bluedevil/bluedevil.pc.in
+++ b/bluedevil/bluedevil.pc.in
@@ -1,5 +1,5 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_PREFIX@/lib
+libdir=@LIB_INSTALL_DIR@
 includedir=@CMAKE_INSTALL_PREFIX@/include
 
 Name: bluedevil
-- 
1.7.9.1

